tests/test_pay.py

```diff
@@ -3218,3 +3218,27 @@ def test_bolt11_null_after_pay(node_factory, bitcoind):
     pays = l2.rpc.listpays()["pays"]
     assert(pays[0]["bolt11"] == invl1)
     assert('amount_msat' in pays[0] and pays[0]['amount_msat'] == amt)
+
+
+def test_listpay_resutl_with_paymod(node_factory, bitcoind):
```

Comment:
```suggestion
def test_listpay_result_with_paymod(node_factory, bitcoind):
```

Reply:
Mh! sorry about that, I apologize to do typing error. This event bring the people to thinks that, I don't pay attention to when I typing (I would think so). But I-m dyslexic and for me is very difficult to find this error.

Thanks for this fix and I will try to pay attention in the future.